summary/table.go

```diff
@@ -23,6 +24,11 @@ var (
 // DisplaySummaryOutput function
 func DisplaySummaryOutput(resp *opb.Response, revDNSLookup bool, requestType string) {
 
+	if len(resp.ProcessData) <= 0 && len(resp.FileData) <= 0 && len(resp.InNwData) <= 0 && len(resp.OutNwData) <= 0 {
+		log.Error().Msgf("no summary data available")
+		return
+	}
```

Comment:
Ideally DisplaySummaryOutput should return the Error, then this error will be handled by Cobra itself!

Reply:
Hello @daemon1024 
Since this is running in a loop, It is not required to throw an error. It is possible to get some values as empty. This error is actually an info.